package.json

```diff
@@ -4,9 +4,9 @@
     "description": "",
     "main": "index.js",
     "scripts": {
-        "lint": "yarn workspaces foreach -t -p -v -i run lint",
-        "test": "yarn workspaces foreach -t -p -v -i run test",
-        "build": "yarn workspaces foreach -t -p -v -i run build",
+        "lint": "yarn workspaces foreach --worktree -t -p -v -i run lint",
+        "test": "yarn workspaces foreach --worktree -t -p -v -i run test",
+        "build": "yarn workspaces foreach --worktree -t -p -v -i run build",
```

Comment:
Complies with https://github.com/yarnpkg/berry/issues/5817#issuecomment-1775047087, what we used previously as default.